cmake/BundledCivetWeb.cmake

```diff
@@ -30,8 +30,8 @@ function(use_bundled_civetweb SOURCE_DIR BINARY_DIR)
     endif()
 
     set(BYPRODUCTS
-            "lib/${PREFIX}civetweb.${SUFFIX}"
-            "lib/${PREFIX}civetweb-cpp.${SUFFIX}"
+            "${CMAKE_INSTALL_LIBDIR}/${PREFIX}civetweb.${SUFFIX}"
```

Comment:
[GNUInstallDirs](https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html) should be included to use CMAKE_INSTALL_* variables on unix systems and CMAKE_INSTALL_LIBDIR should be set explicitly on Windows.

Reply:
Updated, now GNUInstallDirs are included when not on windows and libdir is set accordingly.